maturity_test.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/PuerkitoBio/goquery"
+)
+
+var (
+	githubApiAuthorizationToken = os.Getenv("GITHUB_API_TOKEN")
+	minimumMaturityDate         = time.Now().AddDate(0, -5, 0)
+)
+
+func TestMaturity(t *testing.T) {
+	doc := goqueryFromReadme(t)
+	doc.Find("body li > a:first-child").Each(func(_ int, s *goquery.Selection) {
+		t.Run(s.Text(), func(t *testing.T) {
+			href, ok := s.Attr("href")
+			if !ok {
+				t.Error("expected to have href")
+			}
+
+			matches := reGithubRepo.FindStringSubmatch(href)
+			if matches == nil {
+				return
+			}
+
+			if len(matches) != 3 {
+				t.Fatalf("failed to extract repo and user from: %s, got [%v]", href, strings.Join(matches, ", "))
+			}
+
+			if err := checkRepositoryMaturity(matches[1], matches[2]); err != nil {
```

Comment:
Ypu could use meaningful variable

```suggestion
			user, repo := matches[1], matches[2]
			if err := checkRepositoryMaturity(user, repo); err != nil {
```